android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/components/NpsAnswer.kt

```diff
@@ -0,0 +1,96 @@
+package co.nimblehq.ic.kmm.suv.android.ui.components
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.nimblehq.ic.kmm.suv.android.R
+import co.nimblehq.ic.kmm.suv.android.ui.theme.Typography
+
+@Composable
+fun NpsAnswer(
+    currentIndex: Int = -1,
+    onIndexChange: (Int) -> Unit,
+) {
+    var selectedIndex by remember { mutableStateOf(currentIndex) }
+    val npsSize = 10
+    val npsHeight = 56.dp
+    val npsWidth = 345.5.dp
+    Column {
+        Row(
+            modifier = Modifier
+                .border(BorderStroke(0.5.dp, Color.White), RoundedCornerShape(10.dp))
+        ) {
+            (0 until npsSize).forEach { index ->
+                val isHighlight = index <= selectedIndex
+                val alpha = if (isHighlight) 1f else 0.5f
```

Comment:
I think we could simplify the looping condition a bit. Could you please give it a try?

```
            for (index in 0 until npsSize) {
                ...
            }
```